src/load_inputs/load_dataframe.jl

```diff
@@ -1,19 +1,83 @@
+function filenotfoundconstant()
+    "FILENOTFOUND"
+end
```

Comment:
I made a function for this 'constant' because it seems less bad than an actual global variable, though I'm not sure why. Maybe this is preferable because the interior can't be set to something else? Should look up best practices.